src/api-endpoints.ts

```diff
@@ -4401,6 +4401,27 @@ type NumberFormat =
 
 type PartialBlockObjectResponse = { object: "block"; id: string }
 
+type ApiColor =
```

Comment:
Interesting that ApiColor is included 🤔 Since it seems like it doesn't seem to have come from your lastest PR.